docker/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/* /tmp/* \
     && apt-get autoremove
 
-RUN curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | REPO_SUFFIX=20180726-1 DO_NOT_INSTALL_CATCH_ALL_CONFIG=true bash \
+RUN curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | REPO_SUFFIX=20180810-1 DO_NOT_INSTALL_CATCH_ALL_CONFIG=true bash \
```

Comment:
Not necessarily related to this PR, but I wonder if it makes sense to add a comment with the agent package version here (e.g., 1.5.35-1)?